test/generators/generator.test.js

```diff
@@ -304,4 +304,57 @@ describe('Generator', () => {
       });
     });
   });
+  describe('creating component and test files wrapped in a folder', () => {
+    const templatePath = '/tmp/components/Dumb.js';
+    const creationPath = '/components';
+    const componentName = 'Example';
+    const args = {
+      templatePath,
+      creationPath,
+      componentName,
+      settings: new MockSettings({
+        sourceBase: './tmp/src',
+        testBase: './tmp/test',
+        wrapFilesInFolders: true
+      })
+    };
+    const generator = new Generator(args);
+
+    beforeEach(() => {
+      sinon.spy(console, 'log');
+    });
+
+    afterEach(() => {
+      console.log.restore();
+    });
+
+    describe('#createComponent', () => {
+      it('renders template and writes it to componentPath', () => {
+        sinon.stub(generator, 'renderTemplate').returns('component file');
+
+        generator.createComponent();
+        const file = fs.readFileSync(generator.componentPath(), 'utf8');
+        expect(console.log.calledOnce).to.be.true;
+        expect(file).to.eq('component file');
+
+        fse.removeSync(generator.componentPath());
+        generator.renderTemplate.restore();
+      });
+    });
+
+    describe('#createTest', () => {
+      it('renders template and write it to testPath', () => {
+        sinon.stub(generator, 'renderTemplate').returns('test file');
+
+        generator.createTest();
+
+        const file = fs.readFileSync(generator.componentTestPath(), 'utf8');
+        expect(console.log.calledOnce).to.be.true;
+        expect(file).to.eq('test file');
+
+        fse.removeSync(generator.componentTestPath());
+        generator.renderTemplate.restore();
+      });
+    });
+  });
```

Comment:
these tests arn't really asserting what the path should be when wrapFiles is set to true.  I would instead look [here](https://github.com/SpencerCDixon/redux-cli/blob/master/test/generators/generator.test.js#L114) at how I'm making assertions on what the componentPath should look like.  You can just make another it block that passes the wrapFIles as true flag into MockSettings and then confirm that the component path has the extra nested folder.

You might want to confirm that it keeps the folder name snake_case/PascalCase/whatever as well for consistency.

Also test generated path: should it match the componentPath with the nested folder or when you keep your comps in folders do you not nest them in test directory as well?